dist/srpm/yggdrasil.spec.in

```diff
@@ -50,7 +51,7 @@ exchanging data with its worker processes through a D-Bus message broker.}
 %global godocs          CONTRIBUTING.md README.md
 
 Name:           yggdrasil
-Release:        %{?autorelease}%{!?autorelease:0%{?dist}}
+Release:        %{?autorelease}%{!?autorelease:1.%{builddate}git%{shortcommit}%{?dist}}
```

Comment:
It sets release always to `1.something`. Will be such package resolved as newer, when downstream package will have release higher then `1`? E.g. Will be `yggdrasil-0.4.5-1.20251031gitabcdef.el10.x86_64.rpm` considered as newer, when downstream will be `yggdrasil-0.4.5-3.el10.x86_64.rpm`?

Reply:
I don't know, actually. What I did in this change is manually recreate the value generated by `%autorelease` in situations where `%autorelease` is not defined. This way both Fedora and EL9 `Release` fields are identical. I think this is more desirable than the current state (where Fedora has `1.20240531gitabcdef.fc41` and EL9 has `0.el9`).

We could consider an additional change to automatically set the integer value to something other than 1, but I would propose we do that in a follow-up PR and do it for both cases (where `%autorelease` is defined and where `%autorelease` is not define).